doc/sphinx-guides/source/installation/config.rst

```diff
@@ -956,6 +957,28 @@ See also these related database settings below:
 - :ref:`:Authority`
 - :ref:`:Shoulder`
 
+.. _doi.baseurlstring:
```

Comment:
Should this be doi.mdcbaseurlstring instead?

Reply:
yep